cgo/kuzzle/destructors.go

```diff
@@ -495,6 +504,15 @@ func kuzzle_free_int_result(st *C.int_result) {
 	}
 }
 
+//export kuzzle_free_date_result
```

Comment:
Why did you add this, I don't see any usage of it? Is it a part of another PR?

Reply:
Yes, it is used there: https://github.com/kuzzleio/sdk-cpp/pull/44/files#diff-ad60d65b34e16a3319296bb5d683acd6R79